libs/components/lists/src/lib/modules/repeater/repeater-item.component.ts

```diff
@@ -287,6 +332,90 @@ export class SkyRepeaterItemComponent
     this.repeaterService.unregisterItem(this);
   }
 
+  @HostListener('keydown', ['$event'])
+  public onKeydown($event: KeyboardEvent) {
+    if (
+      $event.target &&
+      ($event.target as Element).matches(
+        [
+          'sky-repeater-item',
+          '.sky-repeater-item',
+          '.sky-repeater-item-left',
+          '.sky-repeater-item-left *',
+          '.sky-repeater-item-title',
+          '.sky-repeater-item-title *',
+        ].join(',')
```

Comment:
Is this array required to avoid capturing events from the `.sky-repeater-item-right` DIV? If so, would it be easier to check that the event does not come from one element, versus checking if the event exists in any number of other elements?

Either way, it would be helpful to include a comment explaining the array's purpose.

Reply:
The idea was to prevent e.g. someone pressing down arrow in an input and that causing focus to change to the next item. I've shifted the code so that might be more obvious.